docs(cli): add set-version to newdocs (#1881)
latticexyz/mud: next-docs/pages/cli/set-version.mdx

# mud set-version

> Set MUD version in all `package.json` files and optionally backup the previously installed version

Reviewer: we no longer do backups in this way

```suggestion
Set MUD dependency version for all packages in the current workspace.
```

## Sample usage

To set the version to the latest `next` release.

```sh copy

> pnpm mud set-version -t next

Reviewer: ```suggestion
pnpm mud set-version --tag next
```

pnpm install
```

## Options

These are the command line options you can specify on `mud set-version`:

| Option         | Meaning                                                      | Type    | Default value |
| -------------- | ------------------------------------------------------------ | ------- | ------------- |
| `--version`    | Show version number                                          | boolean | false         |
| `--mudVersion` | Set MUD to the given version                                 | string  | true          |
| `--tag`        | Set MUD to the latest version with the given tag from npm    | string  |               |

> | `--commit`     | Set MUD to the version based on a given git commit hash from | string  |

Reviewer 2: ```suggestion
| `--commit`     | Set MUD to the version based on a given git commit hash | string  |
```
